src/Loading/babylon.sceneLoader.ts

```diff
@@ -1,16 +1,27 @@
 ﻿module BABYLON {
+    export interface ISceneLoaderPluginExtensions {
+        [extension: string]: {

```

Comment:
For example, glTF can have two extensions (.gltf and .glb) where .gltf is a simple JSON file, and glb a binary version of the JSON file containing binary data such as the vertex/indices buffers etc.